gallery/src/components/NxTransferListHalf/NxTransferListHalfDisableTransferExample.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { DataItem, NxFontAwesomeIcon, NxTransferListHalf } from '@sonatype/react-shared-components';
+import { map, range, prepend } from 'ramda';
+import { faArrowsAltH } from '@fortawesome/free-solid-svg-icons';
+
+const Items: DataItem<number>[] = prepend(
```

Comment:
There's no reason this should start uppercase

```suggestion
const items: DataItem<number>[] = prepend(
```